region/client_test.go

```diff
@@ -942,6 +958,131 @@ func TestSanity(t *testing.T) {
 	wg.Wait()
 }
 
+func TestSanityCompressor(t *testing.T) {
+	ctrl := test.NewController(t)
+	defer ctrl.Finish()
+	mockConn := mock.NewMockConn(ctrl)
+	c := &client{
+		conn:          mockConn,
+		rpcs:          make(chan hrpc.Call),
+		done:          make(chan struct{}),
+		sent:          make(map[uint32]hrpc.Call),
+		rpcQueueSize:  1, // size one to skip sendBatch
+		flushInterval: 1000 * time.Second,
+		compressor:    &compressor{Codec: mockCodec{}},
+	}
+
+	var wg sync.WaitGroup
```

Comment:
Nit: Only ever `Add()`ed to once, you could just use channel that gets closed.

Reply:
used on line 1046 as well